ixbrlparse/core.py

```diff
@@ -198,7 +203,12 @@ def _get_numeric(self) -> None:
                         text=s.text,
                         context=self.contexts.get(s["contextRef"], s["contextRef"]),
                         unit=self.units.get(s["unitRef"], s["unitRef"]),
-                        **s.attrs
+                        ixt=self.namespaces.get(
+                            f'xmlns:{s.attrs["format"].split(":")[0]}', []
+                        )[0]
+                        if s.get("format") is not None
+                        else "",
```

Comment:
This needs to be a bit more robust for the case where it's not found in `self.namespaces` otherwise we'll get lots of keyerrors.